11ty/definitions/spiritual-bypassing.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Spiritual Bypassing
+slug: spiritual-bypassing
+defined: true
+excerpt: a form of appropriation where non-Black individuals use memes and imagery featuring Black people and dark-skinned emojis
+Speech: noun
+reading:
+  - text: What Is Spiritual Bypassing? - Psychology Today
+    href: https://www.psychologytoday.com/us/blog/the-empowerment-diary/201901/what-is-spiritual-bypassing
+  - text: Avoid spiritual bypassing — Anti-Racism Daily
+    href: https://www.antiracismdaily.com/archives/spiritual-bypassing-anti-racism-daily
+  - text: “Spiritual bypassing” is the “all lives matter” of the yoga world. Don't buy it
+    href: https://www.mic.com/p/spiritual-bypassing-is-the-all-lives-matter-of-the-yoga-world-dont-buy-it-27631880
+---
+
+Also referred to as toxic positivity. Spiritual bypassing is behavior that uses spiritual ideas and practices to delay or ignore acknowledging hardship. The pursuit of “good vibes only” or “oneness” can also downplay difficulties brought up by others, especially from [minoritized](/definitions/minoritised/) groups. Spiritual bypassing is present in many Western wellness and self-help practices, primarily Yoga.
```

Comment:
```suggestion
a behavior that uses spiritual ideas and practices to delay or ignore acknowledging hardship; the pursuit of “good vibes only” or “oneness” that can downplay difficulties brought up by others, especially from [minoritized](/definitions/minoritised/) groups; present in many Western wellness and self-help practices, primarily Yoga
```

Reply:
I provided a heavy-handed edit here with the definition because it restates parts of the word; generally we treat this like a "dictionary" definition with broken-out phrases.

One area that needs rewrite is in the last clause starting with "present in..." The way it's written implies that Yoga is a Western wellness practice. If we're going to say something to that effect, and I understand where you're going with it, we need to address that it's the Western and white culture appropriation of Yoga. Without that delineation, it reads that we think Yoga is a white and Western practice.

Reply:
Thank you for this edit and its feedback, I think it reads way better now. I also agree with your comment about the last clause. I'll work to update it.